SPECS/cmake/cmake.spec

```diff
@@ -29,6 +29,10 @@ Patch14:        CVE-2023-27538.patch
 Patch15:        CVE-2023-27535.patch
 Patch16:        CVE-2023-23916.patch
 Patch17:        CVE-2023-46218.patch
+Patch18:        CVE-2024-2398.patch
+Patch19:        CVE-2024-28182.patch
+Patch20:        CVE-2024-7264-1.patch
```

Comment:
This might cause issues with CVE scanning tools (having -1 and -2 patch files). Can you combine them to a single file?

Reply:
Combined the two patches into one file

Reply:
Thank you!